packages/app/src/domain/international/variants-stacked-area-tile/variants-stacked-area-tile.tsx

```diff
@@ -0,0 +1,208 @@
+import { VariantChartValue } from '~/static-props/variants/get-variant-chart-data';
+import { colors } from '~/style/theme';
+import { assert } from '~/utils/assert';
+import { useList } from '~/utils/use-list';
+
+type VariantsStackedAreaTileProps = {
+  values?: VariantChartValue[] | null;
+  metadata: MetadataProps;
+  children?: ReactNode;
+  noDataMessage?: ReactNode;
+};
+
+export function VariantsStackedAreaTile({
+  values,
+  metadata,
+  children = null,
+  noDataMessage = '',
+}: VariantsStackedAreaTileProps) {
+  const { siteText } = useIntl();
+  const text = siteText.internationaal_varianten.varianten_over_tijd_grafiek;
+
+  if (!isPresent(values)) {
+    return (
+      <ChartTile
+        title={text.titel}
+        description={text.toelichting}
+        metadata={metadata}
+      >
+        {children}
+        <NoDataBox>{noDataMessage}</NoDataBox>
+      </ChartTile>
+    );
+  }
+  return (
+    <VariantStackedAreaTileWithData values={values} metadata={metadata}>
+      {children}
+    </VariantStackedAreaTileWithData>
+  );
+}
+
+const alwayEnabled: string[] = [];
+
+type VariantStackedAreaTileWithDataProps = {
+  values: VariantChartValue[];
+  metadata: MetadataProps;
+  children?: ReactNode;
+};
+
+function VariantStackedAreaTileWithData({
+  values,
+  metadata,
+  children = null,
+}: VariantStackedAreaTileWithDataProps) {
+  const { siteText } = useIntl();
+  const { list, toggle, clear } = useList<string>(alwayEnabled);
+
+  const text = siteText.internationaal_varianten.varianten_over_tijd_grafiek;
+  const [seriesConfig, otherConfig, selectOptions] = useSeriesConfig(
+    text,
+    values
+  );
+  const filteredConfig = useFilteredSeriesConfig(
+    seriesConfig,
+    otherConfig,
+    list
+  );
+
+  /* Static legend contains only the inaccurate item */
+  const staticLegendItems: LegendItem[] = [
+    {
+      shape: 'square',
+      color: colors.data.underReported,
+      label: text.legend_niet_compleet_label,
+    },
+  ];
+
+  return (
+    <ChartTile
+      title={text.titel}
+      description={text.toelichting}
+      metadata={metadata}
+      timeframeOptions={['all', '5weeks']}
+    >
+      {(timeframe) => (
+        <>
+          {children}
+          <InteractiveLegend
+            helpText={text.legend_help_tekst}
+            selectOptions={selectOptions}
+            selection={list}
+            onToggleItem={toggle}
+            onReset={clear}
+          />
+          <TimeSeriesChart
+            accessibility={{
+              key: 'variants_stacked_area_over_time_chart',
+            }}
+            values={values}
+            timeframe={timeframe}
+            seriesConfig={filteredConfig}
+            disableLegend
+            dataOptions={{
+              isPercentage: true,
+              forcedMaximumValue: 100,
+            }}
+            formatTooltip={(context) => {
+              /**
+               * With the current context the other_percentage is the first in to be displayed.
```

Comment:
```suggestion
               * With the current config the other_percentage is the first in to be displayed.
```

I think it would be useful to explain why you can't solve this by defining the configuration in the preferred order. I'm guessing other == 100% and can therefor would obscure the rest of the series if it were rendered on top. But if everything is rendered stacked on top of each other, like I would expect this chart to do, then I none of the series should ever overlap each other...

Reply:
I'll have to check with @VWSCoronaDashboard14 , he wrote this part. But I think you're right in your assumption.

Reply:
Ok, checked and changed the comment to better clarify the reasoning behind this.